### ISSUE TITLE
The draggable component item is not keyboard accessible. Users who cannot use a mouse won't be able to add components to the canvas. Consider adding keyboard support with appropriate ARIA attributes (role="button", tabindex="0") and keyboard event handlers for Enter/Space keys to add components.

### ISSUE THREAD

**Issue by sblake94** [sblake94/Juce-Project-Manager-and-Initializer#26]
The draggable component item is not keyboard accessible. Users who cannot use a mouse won't be able to add components to the canvas. Consider adding keyboard support with appropriate ARIA attributes (role="button", tabindex="0") and keyboard event handlers for Enter/Space keys to add components.

_Originally posted by @Copilot in https://github.com/sblake94/Juce-Project-Manager-and-Initializer/pull/24#discussion_r2656641666_
